test/TestCommandLineInvocationSameAsConfig.py

```diff
@@ -22,6 +22,8 @@ def base_arguments():
                           "test/fixtures/skip-tags.yml"),
                          (["--exclude", "test/"],
                           "test/fixtures/exclude-paths.yml"),
+                         (["--show-relpath"],
+                          "test/fixtures/show-relpath.yml"),
```

Comment:
Seen your [comment](https://github.com/ansible/ansible-lint/pull/573#discussion_r402256100) but I am not sure how to not make a mess of this, while placing the files content in the code.

Reply:
Well, this https://github.com/ansible/ansible-lint/blob/master/test/TestIncludeMissingFileRule.py#L9-L16 looks like a good solution to me. What kind of mess are you afraid of?